{"cveId": "CVE-2018-11412", "cwe": ["CWE-416", "CWE-416"], "cvss": [{"baseScore": "4.3", "version": "2.0", "Attack Vector": "None", "Attack Complexity": "M", "Privileges Required": "None", "User Interaction": "None", "Scope": "None", "Confidentiality Impact": "None", "Integrity Impact": "None", "Availability Impact": "P"}], "references": ["https://usn.ubuntu.com/3752-2/", "http://www.securityfocus.com/bid/104291", "https://bugs.chromium.org/p/project-zero/issues/detail?id=1580", "https://usn.ubuntu.com/3752-3/", "https://bugzilla.kernel.org/show_bug.cgi?id=199803", "https://www.exploit-db.com/exploits/44832/", "https://access.redhat.com/errata/RHSA-2019:0525", "https://usn.ubuntu.com/3752-1/"], "description": ["In the Linux kernel 4.13 through 4.16.11, ext4_read_inline_data() in fs/ext4/inline.c performs a memcpy with an untrusted length value in certain circumstances involving a crafted filesystem that stores the system.data extended attribute value in a dedicated inode."], "published": "2018-05-24T18:29:00.190", "state": "PUBLIC", "vendorName": ["n/a"], "productName": ["n/a"], "github": {"advisories": [], "commits": [], "pocAdvisorie": null, "repo": null, "info": {}}, "pocList": []}